### PR TITLE
Refactor some tests in spec folder

### LINKED ISSUE #683: Recfatoring tests in spec folder
Looking for test codes in files from `spec folder` that can be refactored with better practice of coding. These are the folder in spec that own test codes:

- Dashboard;
- Decorators;
- Factores_spec;
- Helpers;

 [Better Specs](https://www.betterspecs.org/) can be helpful to guide you in doubts.

### PR DESCRIPTION
**Pull request by philipeleandro**
fix #683 

@edimossilva @PolinaStamenova 

If there's something to change or that I can improve, please tell me.

### CODE REVIEW

File: backend/spec/factories_spec/answers_spec.rb [hunk 1]
```diff
@@ -4,14 +4,8 @@
...
-
-    it { expect(Answer.count).to eq(1) }
-
-    it { expect(Option.count).to eq(1) }
-
-    it { expect(AnswersSurvey.count).to eq(1) }
+    [Survey, Question, Answer, Option, AnswersSurvey].each do |model|
+      it { expect(model.count).to eq(1) }
+    end
   end
```
Comment (PolinaStamenova) [inline backend/spec/factories_spec/answers_spec.rb:10]: Very nice!

**Review by PolinaStamenova — APPROVED** [review]
Good job @philipeleandro.
I think that nothing else can be refactor for now.

File: backend/spec/factories_spec/answers_spec.rb (hunk 1 above)
Comment (edimossilva) [reply backend/spec/factories_spec/answers_spec.rb:10]: Very nice!

**Review by edimossilva — APPROVED** [review]
nice

**Pull request merged**